app/models/efile_submission.rb

```diff
@@ -47,14 +47,24 @@ def imperfect_return_resubmission?
   end
 
   def generate_irs_address
-    address_service = StandardizeAddressService.new(intake)
-    if address_service.valid?
+    if Rails.env.development?
```

Comment:
@caitbot I was doing the same call over and over again and didn't want to hit the USPS api too many times, so I stubbed this out for development. There's probably a better pattern here... like maybe a config variable that gets passed to the IRS service itself? But wanted to check in whether you thought that stubbing calls to the API in dev is a good idea.